### DISCUSSION TITLE
Drag & Drop not working properly?

### DISCUSSION THREAD

**Discussion by nicholasjativa** [perspective-dev/perspective#1585] [General]
Hello! I'm using the latest version Perspective with Angular. I have it coming in via the CDN and most everything seems to work properly. However, one pretty tough issue that I've not been able to solve is the drag and drop functionality not working. For example, in the Datagrid, I'm able to pick a column from the left-hand side and drag it to the "Group By/Split By/etc" area, however, I'm never actually able to drop it into the area. The area in which the "plus" cursor appears is extremely small. I get no errors or any messages on the console at all.  Sadly, I can't even reproduce this issue in JSFiddle/etc. Is there any insight anyone can give on trying to get it to work? Thank you.
I'm basically doing a simple perspective.worker(), worker.table([someData]), and perspectiveEl.load(table).

**Comment by sc1f** [comment]
What browser and OS are you on? If you're able to try in Chrome, there should be no drag/drop issues. We can work on a browser-specific fix if necessary.

**Comment by nicholasjativa** [comment]
Sorry, I forgot to mention that I'm on Chrome on Windows, latest version I believe (Version 93.0.4577.82) and that this issue literally only occurs in my application (which is evidenced to me by the fact that all of the demos on the Perspective homepage work fine for me, as well as a few JSFiddles that I've made). This leads me to believe that it is something within my app that's causing this, unfortunately. I created a new Angular application from scratch and everything worked fine, so my theory is pretty much confirmed. I'll keep digging. Thank you, however!
